src/components/structures/RoomView.js

```diff
@@ -305,6 +305,20 @@ module.exports = React.createClass({
     _shouldShowApps: function(room) {
         if (!BROWSER_SUPPORTS_SANDBOX) return false;
 
+        // Check if user has prompted to close this app before
```

Comment:
Can you please modify this comment to:

 > // Check if user has previously chosen to hide the app drawer for this room